repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml

```diff
@@ -115,8 +115,8 @@
                         <!-- for a not-current editor the current editor (holder) is reported through the 'inUseBy' feedback -->
                         <hippo:feedback key="inUseBy" value="holder"/>
                     </if>
-                    <if cond="!editor and workflowContext.isGranted(draft,'hippo:admin')">
```

Comment:
Assume removing this doesn't affect admin unlock operation?

Reply:
it does :) hence admin in "unlocker" group by default